spring-cloud-starter-tencent-polaris-ratelimit/src/main/java/com/tencent/cloud/polaris/ratelimit/filter/QuotaCheckServletFilter.java

```diff
@@ -102,6 +102,7 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
 			QuotaResponse quotaResponse = QuotaCheckUtils.getQuota(limitAPI,
 					localNamespace, localService, 1, labels, request.getRequestURI());
 
+			LOG.debug("The request of [{}] will waiting for {}ms.", request.getRequestURI(), quotaResponse.getWaitMs());
```

Comment:
The log must add before thread sleeping.

<img width="834" alt="image" src="https://user-images.githubusercontent.com/24446200/177912891-447115b4-f3fe-4783-94a8-7a6da1eb78a9.png">